pjlib/include/pj/sock.h

```diff
@@ -1527,6 +1527,25 @@ PJ_DECL(pj_status_t) pj_sock_shutdown( pj_sock_t sockfd,
 PJ_DECL(char *) pj_addr_str_print( const pj_str_t *host_str, int port, 
 				   char *buf, int size, unsigned flag);
 
+/**
+ * Create socket pair
+ * @param family    Specifies a communication domain; this selects the
+ *		    protocol family which will be used for communication.
+ * @param type	    The socket has the indicated type, which specifies the 
+ *		    communication semantics.
+ * @param protocol  Specifies  a  particular  protocol  to  be used with the
+ *		    socket.  Normally only a single protocol exists to support 
+ *		    a particular socket  type  within  a given protocol family, 
+ *		    in which a case protocol can be specified as 0.
+ * @param sv	    The new sockets vector
+ *
+ * @return	    Zero on success.
+ *
+ */
+PJ_DECL(pj_status_t) pj_sock_socketpair(int family,
+				    int type,
+				    int protocol,
+				    pj_sock_t sv[2]);
```

Comment:
Is the indentation of 2nd, 3rd, 4th arguments wrong, or is it just my browser?

Reply:
Could you share code formatting rule?

Reply:
I don't we have one IIRC. But as you can see in the function declarations/defs, usually arguments are aligned vertically